profiles/amity-ak97.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/54621148/173427837-88c62bfa-34c4-4228-9103-5cc201514d21.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/173429345-e8914595-0281-4f1b-bded-afea0f6db842.png",
```